### ISSUE TITLE
Allow pool mining based from wallet GUI

### ISSUE THREAD

**Issue by TnTBass** [VeriumReserve/verium#135]
Pool mining currently is exclusive to running a cli miner.  

Benefits:
Any user of an official verium wallet could mine on any pool without the need to run an alternative mining application.
More mining adoptions.  Solo mining is great, but pool mining gets rewards faster, or at least more consistently.  
Ease of use.  Some users will never be comfortable running a cli application.  Efforts should be made to make this coin as easy to use as possible, as that will drive adoptation.

**Comment by effectsToCause** [comment]
I agree this should be a priority feature for next release.  This also would be a necessary feature for when mining shares are live on the network as well.  Where the user has a dropdown and they select mining shares, pool or solo to mine from.
